core/src/test/scala/org/broadinstitute/dsde/rawls/entities/compact/CompactEntityProviderSpec.scala

```diff
@@ -800,7 +800,90 @@ class CompactEntityProviderSpec extends TestDriverComponentWithFlatSpecAndMatche
     actual shouldBe a[EntityNotFoundException]
   }
 
-  "listEntities" should "have tests" is pending
+  behavior of "listEntities"
```

Comment:
because the implementation of `listEntities` is so thin, I don't think tests in this class make sense - you're really only testing how well you configured the mocks. `CompactEntityProviderE2ESpec` would be better, since those tests use the database instead of mocks.